wrappers/opencv/dnn/rs-dnn.cpp

```diff
@@ -3,6 +3,7 @@
 // Please see https://github.com/opencv/opencv/blob/master/LICENSE
 
 #include <opencv2/dnn.hpp>
+#include <opencv2/highgui/highgui_c.h>
```

Comment:
C API is deprecated. Better replace everywhere
cvGetWindowHandle(window_name)
with
getWindowProperty(window_name, WND_PROP_AUTOSIZE) >= 0
to avoid future incompatibilities

Reply:
how could I replace the following code.
HWND hWnd = (HWND) cvGetWindowHandle(windowname.c_str());

I need reference to the opencv window so that i can assign it to a picture control in mfc.